jerry-core/parser/js/js-lexer.c

```diff
@@ -304,42 +304,51 @@ typedef struct
   lexer_token_type_t type;  /**< keyword token type */
 } keyword_string_t;
 
+/**
+ * Keyword list data.
+ */
+typedef struct
+{
+  const keyword_string_t *keyword_p; /**< keyword string list */
+  const uint8_t length;  /**< keyword list length */
+} keyword_string_list_t;
```

Comment:
It is true, that 8 bit length is more than enough, but the first member is a pointer, so the whole structure will be pointer aligned (so we waste 3 byte with 32 bit pointers). What about creating two arrays:
```
static const keyword_string_t * const keyword_strings_list[]
static const uint8_t keyword_lengths_list[]
```
(Note: an s added after string)

Reply:
It seems reasonable. Good idea!